README.md

```diff
@@ -149,7 +149,7 @@ They require JetPack 4.2 and above, and [@dusty-nv](https://github.com/dusty-nv)
 
 ### From Source
 
-If you are installing from source, you will need Python 3.6 or later and a C++14 compiler. Also, we highly recommend installing an [Anaconda](https://www.anaconda.com/distribution/#download-section) environment.
+If you are installing from source, you will need Python 3.6.2 or later and a C++14 compiler. Also, we highly recommend installing an [Anaconda](https://www.anaconda.com/distribution/#download-section) environment.
```

Comment:
The minimum version supported is `3.6.2` in `setup.py`.

Reply:
This actually will cause build failure on current master with Python < 3.6.2, due to the usage of [typing.NoReturn](https://docs.python.org/3/library/typing.html#typing.NoReturn) which was first introduced in 3.6.2